[2025-04-30T00:17:59.690+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2025-04-29T00:00:00+00:00 [queued]>
[2025-04-30T00:17:59.784+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2025-04-29T00:00:00+00:00 [queued]>
[2025-04-30T00:17:59.794+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-04-30T00:17:59.795+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-04-30T00:17:59.797+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-04-30T00:18:00.007+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): update_dataset_2> on 2025-04-29 00:00:00+00:00
[2025-04-30T00:18:00.149+0000] {standard_task_runner.py:55} INFO - Started process 12901 to run task
[2025-04-30T00:18:00.219+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'producer', 'update_dataset_2', 'scheduled__2025-04-29T00:00:00+00:00', '--job-id', '305', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmpnl78x7rm']
[2025-04-30T00:18:00.247+0000] {standard_task_runner.py:83} INFO - Job 305: Subtask update_dataset_2
[2025-04-30T00:18:00.913+0000] {task_command.py:376} INFO - Running <TaskInstance: producer.update_dataset_2 scheduled__2025-04-29T00:00:00+00:00 [running]> on host fae2b1eff5d6
[2025-04-30T00:18:01.754+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=producer
AIRFLOW_CTX_TASK_ID=update_dataset_2
AIRFLOW_CTX_EXECUTION_DATE=2025-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-29T00:00:00+00:00
[2025-04-30T00:18:01.819+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-04-30T00:18:01.903+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2025-04-30T00:18:02.094+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=producer, task_id=update_dataset_2, execution_date=20250429T000000, start_date=20250430T001759, end_date=20250430T001802
[2025-04-30T00:18:02.676+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-04-30T00:18:03.466+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
